idea/idea-completion/src/org/jetbrains/kotlin/idea/completion/KotlinCompletionContributor.kt

```diff
@@ -344,6 +343,15 @@ class KotlinCompletionContributor : CompletionContributor() {
 
                 super.handleInsert(context)
             }
+
+            private fun getToken(file: PsiFile, document: Document, startOffset: Int): PsiElement {
```

Comment:
Minor note: the second parameter could probably be generalized to `document.charsSequence`

Reply:
Thank you, fixed